EventsExpress/Mapping/AutoMapperProfile.cs

```diff
@@ -205,8 +213,15 @@ public AutoMapperProfile()
                 })));
 
             CreateMap<Message, MessageDto>().ReverseMap();
-                
+
+            #endregion
+
+            #region
```

Comment:
Consider naming the region if you are using one.

Reply:
added name of region